godot/tests/docs.rs

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+use godot::prelude::*;
+
+#[derive(GodotClass)]
+#[class(base=Node)]
+/// *documented* ~ **documented** ~ [AABB] [pr](https://github.com/godot-rust/gdext/pull/748)
+pub struct X {
+    /// truly the documentest.
+    #[var]
+    item: f32,
+    base: Base<Node>,
+}
```

Comment:
Could you name this differently than `X`? If we at some point move this to an integration test, it will be in the global namespace of all integration test classes.